CHANGELOG.rst

```diff
@@ -2,13 +2,20 @@
 Changelog
 =========
 
-* Add formatting of filters in variables:
+* Add formatting of variables:
 
   .. code-block:: diff
 
       -{{ egg | crack }}
       +{{ egg|crack }}
 
+* Add formatting of block tags:
+
+  .. code-block:: diff
+
+      -{% if  breakfast  ==  'scrambled eggs'  %}
```

Comment:
I believe this is space before the if you are formatting for isn't it ? 

```rust
#[test]
fn test_format_block_bits() {
    let formatted = format("{%  if breakfast  ==  'egg'  %}\n", None);
    assert_eq!(formatted, "{% if breakfast == 'egg' %}\n");
}
```

Reply:
Sorry, what do you mean?